use-timescale/metrics-logging/metrics-to-prometheus.md

```diff
@@ -0,0 +1,73 @@
+---
+title: Export metrics to Prometheus
+excerpt: Export telemetry metrics to Prometheus
+products: [cloud]
+keywords: [integration, metrics, Prometheus, alerting]
+tags: [telemetry, monitor]
+cloud_ui:
+    path:
+        - [integrations]
+        - [services, :serviceId, operations, integrations]
+---
+
+import ExporterRegionNote from 'versionContent/_partials/_cloud-integrations-exporter-region.mdx';
+
+
+# Export metrics to Prometheus
+
+## Create a monitoring user
+
+1. Connect to the Timescale service as tsdbadmin (using psql or a similar tool).
+
+1. Create a new user to scrape the metrics from the database using the required permissions:
+
+    a. Create a user named `monitoring` with the desired password by replacing `<password>`:
+    
+    ```sql
+    CREATE USER monitoring WITH PASSWORD '<password>';
+    ```
+
+    b. Grant `pg_read_all_stats` to the `monitoring` user:
+
+    ```sql
+    GRANT pg_read_all_stats to monitoring;
+    ```
+
+## Install PostgreSQL exporter
+
+1. Install the postgresql_exporter on a host that you manage to connect to the Timescale service and collect telemetry. Below is an example using Docker to run postgresql_exporter:
+
+    <ExporterRegionNote />
+
+    make sure to replace `<PASSWORD>` with the created password and `<TIMESCALE-HOST:PORT>` with the timescale service host and port.
+
+    ```bash
+    docker run \
+    -p 9187:9187 \
+    -e DATA_SOURCE_NAME="postgres://monitoring:<PASSWORD>@<TIMESCALE-HOST:PORT>/tsdb?sslmode=require" \
+    quay.io/prometheuscommunity/postgres-exporter
+    ```
+
+1. Once the postgresql_exporter is up and running, and successfully connected to the Timescale service, you can configure your Prometheus server to scrape the postgresql_exporter metrics endpoint. This endpoint exposes all the metrics provided by the exporter. 
```

Comment:
Do we know how many connections will be consumed by this exporter?

Reply:
This is proportional to the number of databases that the exporter is scraping. Ideally, if the exporter is configured to scrape from one database, it should limit the connections to just one, as shown below:

![Screenshot 2023-10-05 at 4 10 47 PM](https://github.com/prometheus-community/postgres_exporter/assets/25104868/dfabb983-ccfa-4668-8afa-da2ed7015fd8)